### PR TITLE
BoxContainer description to point to the "Using Containers" page

### PR DESCRIPTION
**Pull request by CheetoMonster**
Add a link to the page discussing Containers in general.

Users happening across this page may not already know the intent of Containers, or which others are available.  Someone has already written a helpful page with that information ("Using Containers"); this change just adds a link to it.

If this edit is acceptable, I can make similar one-line changes to the other ten Container-derived classes that may benefit.

### CODE REVIEW

**Comment by Calinou** [comment]
The change seems useful to me, just make sure to follow the suggestion I made when adding this link to the other Container class pages.

**Comment by akien-mga** [comment]
The change to use `$DOCS_URL` instead of hardcoding the website is still needed. It would also be good to amend the commit message to be more explicit about what the commit does.

**Comment by CheetoMonster** [comment]
Hey Remi,

I wondered about the use of "latest" in the URL.  To clarify, should I
change this:
  <link title="GUI containers">
https://docs.godotengine.org/en/latest/tutorials/ui/gui_containers.html
</link>
to this:
  <link title="GUI
containers">$DOCS_URL/tutorials/ui/gui_containers.html</link>
?

(Searching godotengine.org for DOCS_URL tells me nothing, so I'm falling
back to a programmer's guess.)

Sorry for mangling the commit message -- I thought the new one would be
appended, not fully overwrite it.  I will fix.

Thanks,
-Chris

**Comment by akien-mga** [comment]
Yes that's correct. `$DOCS_URL` is replaced by the base URL, the language code and the version.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

File: doc/classes/BoxContainer.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Arranges child [Control] nodes vertically or horizontally, and rearranges them automatically when their minimum size changes.
+		See also: <link title="An overview of containers">https://docs.godotengine.org/en/latest/tutorials/ui/gui_containers.html</link>
 	</description>
 	<tutorials>
 	</tutorials>
```
[the remark this answers is not in the document]
Comment (gokhankerim) [reply doc/classes/BoxContainer.xml]: bro help me

**Comment by akien-mga** [comment]
Cherry-picked for 3.5.

**Pull request merged**